<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mine_gas_flow_predict_dlg</class>
 <widget class="QWidget" name="mine_gas_flow_predict_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>矿井相对瓦斯涌出量预测</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../usecad.qrc">
    <normaloff>:/images/cbm.ico</normaloff>:/images/cbm.ico</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>生产采区采空区瓦斯涌出系数K'</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="K1_gas">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="k1_help">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>帮 助</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>已采采区采空区瓦斯涌出系数K''</string>
   </property>
  </widget>
  <widget class="QPushButton" name="k2_help">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>帮 助</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="K2_gas">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>&gt;&gt;生产采区参数</string>
   </property>
   <widget class="QPushButton" name="work_area_cacl">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>计 算</string>
    </property>
   </widget>
   <widget class="QPushButton" name="assign">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>布 置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>开采煤层</string>
    </property>
   </widget>
   <widget class="QComboBox" name="coal">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>平均日产量</string>
    </property>
   </widget>
   <widget class="QComboBox" name="work_area">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="work_area_qr">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="design_work_area">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>设 计</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="work_area_A">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>生产采区</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>相对瓦斯涌出量</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>&gt;&gt;矿井相对瓦斯涌出量预测结果</string>
   </property>
   <widget class="QPushButton" name="cacl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>预 测</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>保 存</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../usecad.qrc"/>
 </resources>
 <connections/>
</ui>
